source/component/PasDoc_Parser.pas

```diff
@@ -898,20 +898,36 @@ procedure TParser.ParseRoutine(out M: TPasRoutine;
     until IsSemicolon and (Level = 0);
   end;
 
+  function IsParameterIdentifier(Token: TToken): Boolean;
+  var
+    Text: string;
+  begin
+    if Token.MyType = TOK_IDENTIFIER then
+      Result := True
+    else if Token.MyType = TOK_KEYWORD then begin
```

Comment:
Put `begin` on the same indentation as `end`, and `if` at the beginning, so

```
    else 
    if Token.MyType = TOK_KEYWORD then 
    begin
       ...
    end else
```

I know, we have various styles in PasDoc, but going forward I'd like to make it consistent -- and I'd say to follow CGE coding style, https://castle-engine.io/coding_conventions .